arviz/stats/diagnostics.py

```diff
@@ -536,7 +536,19 @@ def _bfmi(energy):
 
 
 def _backtransform_ranks(arr, c=3 / 8):
-    """Backtransformation of ranks."""
+    """Backtransformation of ranks.
```

Comment:
I guess dumb question still but what does this actually do again? Either math or practical wise?

Reply:
old normalization

    (rank - 0.5) / size)

new normalization (c=3/8)

    (rank- c) / (size - 2 * c + 1)

So with c=4/8 they are the same

Reply:
According to Blom (1958), using fraction 3/8 gives values that match better the expected ordered statistics. This probably doesn't have any practical significance for convergence diagnostics but as the paper reviewer was not familiar with rank normalization, we had to go with the fraction that has been agreed to be the most accurate one.